docs/source/compute_config/aws_lambda.md

```diff
@@ -44,34 +43,72 @@ python3 -m pip install lithops[aws]
 
 7. Choose **Lambda** on the use case list and click **Next: Permissions**. Select the policy created before (`lithops-policy`). Click **Next: Tags** and **Next: Review**. Type a role name, for example `lithops-execution-role`. Click on *Create Role*.
 
-## Configuration
+## AWS Credential setup
 
-6. Edit your lithops config and add the following keys:
+Lithops loads AWS credentials as specified in the [boto3 configuration guide](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html).
 
+In summary, you can use the following settings:
+
+1. Provide credentials via the `~/.aws/config` file. **This is the preferred option to configure AWS credentials for use with Lithops**:
+
+    You can run `aws configure` command if the AWS CLI is installed to setup the credentials.
+
+2. Provide credentials via environment variables:
+
+    Lithops needs at least `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and `AWS_DEFAULT_REGION` environment variables set.
```

Comment:
Maybe in this option you can put a config example (and maybe remove `AWS_DEFAULT_REGION`?):

lithops:
    backend: aws_lambda

aws_lambda:
    execution_role: <EXECUTION_ROLE_ARN>
    region: <REGION_NAME>